modules/tfm/tfm/boards/common/plat_init.c

```diff
@@ -16,7 +16,11 @@
 enum tfm_hal_status_t tfm_hal_platform_init(void)
 {
 	__enable_irq();
+
+	/* If TF-M does not use the UART1 then do not initialize it */
+	#ifdef CONFIG_TFM_SECURE_UART1
 	stdio_init();
+	#endif
```

Comment:
Comment reads "if doesn't use then don't initialize."

But code reads "if uses then initialize".

It's clearer if we write

`// Only if UART1 is used by TF-M do we initialize it.
`

or

```
#else
// If TF-M does not use the UART1 then do not initialize it
#endif

```

But some static analyzers and human analyzers don't like the last form.

Nitpicking, so don't fix unless other patches are needed anyway.

Reply:
Had to rebase, so this is done